.travis.yml

```diff
@@ -5,44 +5,58 @@ node_js:
   # so that your addon works for all apps
   - '8'
 
-sudo: required
 dist: trusty
 
 addons:
   chrome: stable
 
 cache:
   yarn: true
+  directories:
+    - node_modules
 
-global:
-  # See https://git.io/vdao3 for details.
-  - JOBS=1
 env:
-  # we recommend new addons test the current and previous LTS
-  # as well as latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-1.12
-  - EMBER_TRY_SCENARIO=ember-1.13
-  - EMBER_TRY_SCENARIO=ember-lts-2.8
-  - EMBER_TRY_SCENARIO=ember-lts-2.18
-  - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
-  - EMBER_TRY_SCENARIO=ember-default
-
-matrix:
-  fast_finish: true
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+branches:
```

Comment:
What is the intent with this?

Reply:
It comes from the [default `travis.yml`](https://github.com/ember-cli/ember-addon-output/blob/master/.travis.yml#L23) that is generated by `ember addon`.

The docs from travis: https://docs.travis-ci.com/user/customizing-the-build/#building-specific-branches
I cannot tell if that means it will avoid building pull-request branches, if so I will remove this setting.

Reply:
Apparently it prevents double-builds for branches pushed by the repo owner, see:
https://github.com/ember-cli/ember-cli/commit/30eab4c1b30976b88b0bdf8c0000832106b7ff44